app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -1722,6 +1722,7 @@ private void handleFormLoadCompletion(FormController fc) {
         try {
             FormEntrySessionReplayer.tryReplayingFormEntry(mFormController.getFormEntryController(),
                     formEntryRestoreSession);
+            formEntryRestoreSession = null;
```

Comment:
Fixes bug where if you rotate the screen after restoring a form replay session it would crash commcare due to trying to replay an empty session